assets/scss/light-theme.scss

```diff
@@ -368,6 +368,10 @@ div.aside div.aside-content {
   background: #D9E4EA !important;
 }
 
+#hero .product-links > .linked-box {
+  background: #D9E4EA !important;
+}
+
```

Comment:
This was the only thing that I found that wasn't rendering well in light mode, though I could have missed something.